tools/imagepuller-eval/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/edgelesssys/contrast/imagepuller-eval
+
+go 1.24.0
+
+replace github.com/edgelesssys/contrast/imagepuller => ../../imagepuller
```

Comment:
This probably shouldn't be required in the workspace...? Could not get go to *not* try and use the (non-existent) remote version from Github otherwise though